File: 0913/MainActivity.java
package com.cookandroid.helloandroid;

import android.content.Intent;
import android.graphics.Color;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Button btnHomepage, btnCall, btnGallery, btnFinish;
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().setDisplayShowHomeEnabled(true);
        setTitle("FourButton");

        btnHomepage = (Button) findViewById(R.id.btnHomepage);
        btnCall = (Button) findViewById(R.id.btnCall);
        btnGallery = (Button) findViewById(R.id.btnGallery);
        btnFinish = (Button) findViewById(R.id.btnFinish);

        btnHomepage.setBackgroundColor(Color.GRAY);
        btnCall.setBackgroundColor(Color.GREEN);
        btnGallery.setBackgroundColor(Color.RED);
        btnFinish.setBackgroundColor(Color.YELLOW);

        btnCall.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                Intent mIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("tel:/911"));
                startActivity(mIntent);
            }
        });
        btnHomepage.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                Intent mIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://m.nate.com"));
                startActivity(mIntent);
            }
        });
        btnGallery.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                Intent mIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("content://media/internal/images/media"));
                startActivity(mIntent);
            }
        });
        btnFinish.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: Project/source/kotlin/MainActivity.kt
package com.example.kotlinapp

import android.app.Activity
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.VideoView
import kotlin.system.exitProcess

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnHome = findViewById<Button>(R.id.btnHome)
        val btnHeal = findViewById<Button>(R.id.btnHealth)
        val btnStr = findViewById<Button>(R.id.btnStretch)
        val btnExit = findViewById<Button>(R.id.btnExit)
        val btnMe = findViewById<Button>(R.id.btnMe)

        btnHome.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .replace(R.id.view, HomeFragment())
                .commit()
        }
        btnHeal.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .replace(R.id.view, HealthFragment())
                .commit()
        }
        btnStr.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .replace(R.id.view, StretchFragment())
                .commit()
        }
        btnMe.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .replace(R.id.view, MeFragment())
                .commit()
        }
        btnExit.setOnClickListener {
            // 현재 액티비티를 종료시킨다.
            finish()
        }
    }

}